Workiva/over_react_codemod
# FED-1919 Codemod to add required flux actions / store prop(s)

## Motivation
The null safety over_react major release will make a breaking change in which `FluxPropsMixin`'s `store` and `actions` props are required, so that they can be made non-nullable.

Making `store` required shouldn't affect most usages, but `actions` will definitely affect some consumers.

While the `actions` prop will be present in usages in most cases, there are usages that only wire up a component to redraw on the store, because the actions aren't used. 

## Changes
1. Create a `RequiredFluxProps()` codemod class that patches invoked and un-invoked builders that mix-in `FluxUiPropsMixin` when either the `actions` or `store` prop(s) are missing.
    - Adds the missing setter with a `null` value unless an in-scope var with matching type is found.
    - If an in-scope var is found with a matching type - that var is set as the value.
1. Write tests for all conceivable patch cases

## QA
1. Verify the changes produced by [this batch](https://github.com/Workiva/sourcegraph-batch-changes/pull/325) that executes this codemod.
